src/material/core/datetime/date-adapter.ts

```diff
@@ -10,20 +10,20 @@ import {inject, InjectionToken, LOCALE_ID} from '@angular/core';
 import {Observable, Subject} from 'rxjs';
 
 /** InjectionToken for datepicker that can be used to override default locale code. */
-export const MAT_DATE_LOCALE = new InjectionToken<string>('MAT_DATE_LOCALE', {
+export const MAT_DATE_LOCALE = new InjectionToken<{}>('MAT_DATE_LOCALE', {
```

Comment:
any reason to not just use unknown?

Reply:
I think that might be a breaking change, I went with `{}` since it's what we discussed on Monday.